src/main.cpp

```diff
@@ -515,6 +518,8 @@ int main(int argc, char **argv)
 					s_bInitialWantsVRREnabled = true;
 				} else if (strcmp(opt_name, "expose-wayland") == 0) {
 					bExposeWayland = true;
+				} else if (strcmp(opt_name, "wm-class") == 0) {
+					setenv( "SDL_VIDEO_X11_WMCLASS", strdup(optarg), 1);
```

Comment:
Is there a better way setting env?

If it's just an env var, why can't users set it instead of adding a new flag to gamescope?

Reply:
sure, I can set the env var directly, but it is rather hard to find (I had to dig into SDL source to figure out how SDL assigns window class hints) so having better discoverability might be worth the trouble. a bunch of X11 applications have this functionality directly exposed so for me this is something to be expected to have, regardless of how it works internally.

anyway, if you don't think this merits its own option, i'm not going to argue. :shrug:

>Is there a better way setting env?

i'm not sure what this means. maybe?

Reply:
There is not a better way of overriding it in SDL.